irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -1286,13 +1286,30 @@ namespace iroha {
           {"createAccount",
            createAccountBase,
            {(boost::format(R"(
-            has_perm AS (%s),)")
+           domain_role_permissions_bits AS (
+                 SELECT COALESCE(bit_or(rhp.permission), '0'::bit(%1%)) AS bits
+                 FROM role_has_permissions AS rhp
+                 WHERE rhp.role_id = (SELECT * FROM get_domain_default_role)),
```

Comment:
Can't this be done with `JOIN` as below? They are more efficient, as far as I know

Reply:
we are not trying here to perform ```SELECT ... FROM a, b WHERE a.id=b.id```. That is why where statement could not be substituted by join.